index.html

```diff
@@ -126,10 +126,22 @@
       </p>
       <section class="informative">
         <h3>
-          Example manifest
+          Example manifests
         </h3>
         <p>
-          The following shows a typical <a>manifest</a>.
+          The following shows a very simple <a>manifest</a>.
+        </p>
+        <pre class="example json" title="typical manifest">
```

Comment:
Do you mean I should update the title of the pull request, the "Example Manifests" title, or a different title?